test/common.js

```diff
@@ -423,8 +423,10 @@ function runCallChecks(exitCode) {
 }
 
 
-exports.mustCall = function(fn, expected) {
-  if (typeof expected !== 'number') expected = 1;
+exports.mustCall = function(fn, expected = 1) {
```

Comment:
I think the simplest thing to do would be to update this PR. If `expected === undefined`, set it to 1. Otherwise, throw a `TypeError` if `typeof expected !== 'number'`. I don't think we need to worry about negative numbers slipping through, because they should blow up during development.